src/OpenTelemetry.Exporter.InfluxDB/InfluxDBExporterExtensions.cs

```diff
@@ -0,0 +1,79 @@
+// <copyright file="InfluxDBExporterExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using InfluxDB.Client;
+using OpenTelemetry.Exporter.InfluxDB;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Metrics;
+
+/// <summary>
+/// Extension methods for <see cref="TracerProviderBuilder"/> for using InfluxDB.
+/// </summary>
+public static class InfluxDBExporterExtensions
+{
+    /// <summary>
+    /// Adds InfluxDB exporter to the TracerProvider.
+    /// </summary>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> builder to use.</param>
+    /// <param name="configure">Callback action for configuring <see cref="InfluxDBMetricsExporterOptions"/>.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+    public static MeterProviderBuilder AddInfluxDBMetricsExporter(this MeterProviderBuilder builder, Action<InfluxDBMetricsExporterOptions> configure)
+    {
+        builder.AddReader(_ =>
+        {
+            var options = new InfluxDBMetricsExporterOptions();
+            configure.Invoke(options);
+
+            var influxDbClientOptions = new InfluxDBClientOptions(options.Endpoint.ToString());
+            if (!string.IsNullOrWhiteSpace(options.Bucket))
+            {
+                influxDbClientOptions.Bucket = options.Bucket;
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.Org))
+            {
+                influxDbClientOptions.Org = options.Org;
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.Token))
+            {
+                influxDbClientOptions.Token = options.Token;
+            }
+
+            var influxDbClient = new InfluxDBClient(influxDbClientOptions);
+            var writeApi = influxDbClient.GetWriteApi(new WriteOptions
+            {
+                FlushInterval = options.FlushInterval,
+            });
+            var metricsWriter = CreateMetricsWriter(options.MetricsSchema);
+            var exporter = new InfluxDBMetricsExporter(metricsWriter, influxDbClient, writeApi);
+            return PeriodicExportingMetricReaderHelper.CreatePeriodicExportingMetricReader(exporter, new MetricReaderOptions(), defaultExportIntervalMilliseconds: 1000);
```

Comment:
The default aggregation temporality for MetricReader is [Cumulative](https://github.com/open-telemetry/opentelemetry-specification/blob/279e5e17aa8307578794c1ded47cfcf2bf5d950a/specification/metrics/data-model.md#temporality:~:text=Contrast%20with%20cumulative%20aggregation%20temporality%20where%20we%20expect%20to%20report%20the%20full%20sum%20since%20%27start%27%20(where%20usually%20start%20means%20a%20process/application%20start)%3A) and not [Delta](The aggregation temporality is used to understand the context in which the sum was calculated. When the aggregation temporality is "delta", we expect to have no overlap in time windows for metric streams, e.g.). Just want to confirm that Cumulative aggregation temporality is what you intend to use for this exporter.

Reply:
I'd follow here [InfluxDB/IOx Common Observability Schema](https://github.com/influxdata/influxdb-observability/blob/main/docs/metrics.md). From my understanding, InfluxDB only supports the Cumulative Aggregation Temporality.